app/frontend/src/components/admin/AdminFormsTable.vue

```diff
@@ -1,24 +1,29 @@
 <template>
-  <div>
+  <div :class="{ 'dir-rtl': isRTL }">
     <v-row no-gutters>
-      <v-col cols="12" sm="8">
+      <v-col cols="6">
         <v-checkbox
           class="pl-3"
+          :class="isRTL ? 'float-right' : 'float-left'"
           v-model="activeOnly"
           :label="$t('trans.adminFormsTable.showDeletedForms')"
           @click="refeshForms"
         />
       </v-col>
-      <v-col cols="12" sm="4">
+      <v-col cols="6">
```

Comment:
I'm curious about the change here of the `cols` and `sm` values. Is this to change the layout on mobile devices so that the deleted forms checkbox and the search field are always on the same line?

Reply:
This is the old:
<img width="167" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/74d9ecba-217d-48ef-803e-909246168067">
and the new:
<img width="170" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/a0b609f0-d7cc-4218-9fe2-e00ce60877ea">